src/components/Campaign.tsx

```diff
@@ -121,7 +122,14 @@ export const Campaign = () => {
             </EditorTopBar>
           </Header>
           <Footer>
-            <EditorBottomBar {...continuationUrls}>
+            {campaignContentMutation.isLoading ? (
+              <SavingMessage>
+                <FormattedMessage id="saving" />
+              </SavingMessage>
+            ) : (
+              false
+            )}
```

Comment:
What do you think about moving the _if_ to the lower layer?

```suggestion
            <SavingMessage visible={campaignContentMutation.isLoading}>
              <FormattedMessage id="saving" />
            </SavingMessage>
```

Reply:
Solved 🚀